int64 option
uint16 anguloMax
uint16 anguloMin
---
string feedback
